.studiorc

```diff
@@ -208,11 +208,16 @@ _stop-builder() {
 stop-builder() { stop-on-failure _stop-builder "$@"; }
 
 generate_bldr_keys() {
-  KEY_NAME=$(hab user key generate bldr | grep -Po "bldr-\d+")
-  for svc in api jobsrv worker; do
-    hab file upload "builder-${svc}.default" $(date +%s) "/hab/cache/keys/${KEY_NAME}.pub"
-    hab file upload "builder-${svc}.default" $(date +%s) "/hab/cache/keys/${KEY_NAME}.box.key"
-  done
+  if ls /hab/cache/keys/bldr-*.pub 1> /dev/null 2>&1; then
```

Comment:
I think it's a bit more idiomatic to write:
```bash
  if [ -f /hab/cache/keys/bldr-*.pub ]; then
```
and it obviates the redirection stuff

Reply:
Unfortunately that doesn't work - if there are multiple files bash expands them all out, then that line fails with an error.   Searching the web for a solution seems like there's a huge variety and nothing ideomatic for this - I just picked one that seemed simple enough.

Reply:
Oh, good call. It's a bummer that there isn't a well supported test construct for this. One other option that I still like a _little_ better is:
```bash
if [ -n "$(find /hab/cache/keys -name "*.pub")" ]; then
```
but either way is really fine.